### ISSUE TITLE
Not enough space - no message

### ISSUE THREAD

**Issue by JindrichPilar** [helmuthdu/aui#229]
When trying to install on disk with not enough space, no error message is shown (or logged). Most following commands also fail without error or with message directory not found.

fifo step 5) Install Base System
- some error message flashes very fast, unreadable. Instalation script says everything is OK.

Isn't really big problem, not user friendly though.

Using
* archlinux-2015.01.01-dual.iso   
* Virtualbox

**Comment by helmuthdu** [comment]
It's a good thing to have, can you try to create a patch? ^^

**Comment by JindrichPilar** [comment]
fixed by https://github.com/helmuthdu/aui/pull/231

However If it should provide nicer more specific message and avoid invoking ``pacstrap`` also adding this might work:

```BASH
check_free_space "${MOUNTPOINT}" 800 
[[ $? -ne 0 ]] && error_msg "Not enough space on ${MOUNTPOINT}"
```

```BASH
#file: sharedfuncs
check_free_space() { #{{{
  free_space=$(df -P -BM "${1}" | tail -1 | tr -s ' ' | cut -d' ' -f4 | cut -dM -f1)

  [[ "${2}" -le "${free_space}" ]] && return 0 || return 1
} #}}}
```                                                                                                                                                                                                                                           

On the other hand it has hardcoded [800M](https://wiki.archlinux.org/index.php/beginners'_guide#System_requirements) free space which is definitue minus. Not sure if not calling ``pacstrap`` is worth it.

**Comment by mibbio** [comment]
Maybe writing an extra check for available disk space is unnecessary work, as it only handles one specific pacstrap error.
Pacstrap by itself has, besides others, a check for target disk space with corresponding messages.
To halt the script on any non zero return value of pacstrap is a more generic solution.

**Comment by JindrichPilar** [comment]
Didn't know ``pacstrap`` checked it, I though it just called pacman. Well in that case there is no reason for adding duplicate check to ``aui``.

**Comment by mibbio** [comment]
Ok, `pacstrap` just calls `pacman`, but then `pacman` does the check before downloading or installing any packages.
The idea of my solution was to get a simple way to catch any possible pacstrap error.
And i thought it's better to preserve the original `pacstrap` messages instead of wrapping everything with own messages.

**Comment by JindrichPilar** [comment]
You are completely right about pacstrap message. I don't have anyproblem with your solution, I think preserving original messages should be there for any program ``aui`` calls.

Now I might be completely wrong but when  calling ``pacman`` it will fetch information about required packages and calculates size. Which on slow internet connection can take a while. And in that case ``aui`` telling me I created partition badly before calling pacman would be much appreciated. 
Which is what I failed to explain when I suggested that free space check with custom script.
